scripts/template/src/patternfly/{folder}/{name}/examples/{name}.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: '{name}'
+title: '{nameSpacerized}'
```

Comment:
Oh oops, can we lower case any extra words? It's currently generating title case and it should be sentence case.

ie, from `node generate FooComponent` generates `Foo Component`, but should generate `Foo component`

Reply:
Done.

Reply:
Perfect!